GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -41,6 +45,8 @@ @implementation GIDGoogleUser {
   OIDAuthState *_authState;
 }
 
+@synthesize configuration = _configuration;
```

Comment:
This works, but it might be clearer to simply define an instance variable above, as we don't want anything else from `@synthesize` here.  Also should consider renaming to _cachedConfiguration or similar to make it obvious what the role of the ivar is.